typings/models.d.ts

```diff
@@ -73,12 +73,14 @@ type DateString = {
   hidden?: boolean;
 };
 
+type SelectValue = { label: string; value: string | number } | string;
```

Comment:
@dlehmhus does this work? typescript `3.5` refused to compile due to the [use of `{ label: string, value: number }` in `model/settings.ts`](https://github.com/cotype/core/blob/master/src/model/settings.ts#L40-L47)

Reply:
That should work! or `{ label: string; value: string | number } | string | number` ?

Reply:
Thought about that, too but since it was `string[]` before I only wanted to make a "minimal" change.